applications/DEMApplication/custom_search/bins_dynamic_objects_periodic.h

```diff
@@ -342,8 +365,8 @@ inline void NextIndex(IndexType& Index, int& box_size_counter, PointType& MinCel
     }
     else {
         Index = 0;
-        MinCell[dimension] += - MinCell[dimension];
-        MaxCell[dimension] += - MaxCell[dimension] + this->mCellSize[dimension];
+        MinCell[dimension] = 0;
```

Comment:
This looks kinda dangerous. Isn't it changing the behaviour of the search?

Reply:
We were subtracting the LHS to the LHS. The result is 0